satpy/readers/insat3d_img_l1b_h5.py

```diff
@@ -182,6 +182,8 @@ def get_area_def(self, ds_id):
         a = 6378137.0
         b = 6356752.314245
 
+        nom_cen_pos = self.datatree.attrs["Nominal_Central_Point_Coordinates(degrees)_Latitude_Longitude"][1]
```

Comment:
Would it be possible to have another variable name, eg `subsatellite_longitude`?